src/App.js

```diff
@@ -1,6 +1,6 @@
 import { Route, Routes } from "react-router-dom";
 import { Footer, Header } from "./components/index";
-import { Homepage, VideoListingpage } from "./pages/index.js";
+import { Homepage, Signin, Signup, VideoListingpage } from "./pages/index.js";
```

Comment:
You don't need to write `./pages/index.js`, instead of that you can directly use `./pages`. It will behave as same.